docs/deploy-cluster.md

```diff
@@ -0,0 +1,144 @@
+---
+
+This tutorial provides a introduction to deploy `Nebula` cluster.
+
+---
+
+#### Download and install package
+
+First at all, you can download rpm or deb from [Here]().
+
+For `CentOS`:
+
+```
+rpm -ivh nebula-{VERSION}-0.el7.centos.x86_64.rpm
+```
+
+For `Ubuntu`:
+
+```
+dpkg -i nebula-{VERSION}-0.el7.centos.x86_64.deb
+```
+
+By default, the config files are under `/usr/local/etc`, you should modify the `meta_server_addrs` to set the Meta Server's address. 
```

Comment:
@laura under `/usr/local/nebula/` or `/usr/local/`

Reply:
Currently it's `/usr/local/`

Reply:
chang to /usr/local/nebula

Reply:
we should introduce the linux system version. now we offer centos7.5,centos6.5,Ubuntu1604 and Ubuntu1804